staging/src/k8s.io/client-go/util/retry/util.go

```diff
@@ -42,21 +42,8 @@ var DefaultBackoff = wait.Backoff{
 	Jitter:   0.1,
 }
 
-// OnError executes the provided function repeatedly, retrying if the server returns a specified
-// error. Callers should preserve previous executions if they wish to retry changes. It performs an
-// exponential backoff.
-//
-//     var pod *api.Pod
-//     err := retry.OnError(DefaultBackoff, errors.IsConflict, func() (err error) {
-//       pod, err = c.Pods("mynamespace").UpdateStatus(podStatus)
-//       return
-//     })
-//     if err != nil {
-//       // may be conflict if max retries were hit
-//       return err
-//     }
-//     ...
-//
+// OnError is an abstraction of RetryOnConflict, which allows the caller to control which
+// errors will result in a retry.
 // TODO: Make Backoff an interface?
 func OnError(backoff wait.Backoff, errorFunc func(error) bool, fn func() error) error {
 	var lastConflictErr error
```

Comment:
Since you are here, could you rename this variable? It shouldn't have "conflict" in the name.